srv/salt/ceph/stage/discovery/default.sls

```diff
@@ -4,9 +4,17 @@ ready:
     - name: minions.ready
     - timeout: {{ salt['pillar.get']('ready_timeout', 300) }}
 
+refresh_pillar0:
+  salt.state:
+    - tgt: {{ salt['pillar.get']('master_minion') }}
```

Comment:
would you need pillar data from other minions during your benchmarking runs... as far as custom roles go since the customizations go in the global/cluster yamls I think it is okay if we just do a refresh in master minion, thoughts?